src/StripeTests/StripeTests.csproj

```diff
@@ -16,8 +16,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="coverlet.msbuild" Version="2.7.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="2.1.0" />
```

Comment:
I removed this dependency. I added it a while ago in #1298 but I refactored the test at some point and it's no longer useful.